apps/molgenis-components/src/components/forms/InputRefList.vue

```diff
@@ -51,14 +52,27 @@
             {{ applyJsTemplate(row, refLabel) }}
           </label>
         </div>
-        <ButtonAlt
-          class="pl-0"
-          :class="showMultipleColumns ? 'col-12 col-md-6 col-lg-4' : ''"
-          icon="fa fa-search"
-          @click="openSelect"
-        >
-          {{ count > maxNum ? `view all ${count} options.` : "view as table" }}
-        </ButtonAlt>
+        <div v-if="canEdit">
+          <Tooltip value="New entry">
+            <RowButtonAdd
```

Comment:
I think we should hid '+' if the refTable is NOT current schema (or find a way to quickly find out if we have permission on that other schema).